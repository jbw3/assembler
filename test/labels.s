# labels

# line after
label1:
	mov r0, r1

# same line
label2 : addi r2, label1

# 2 lines after
label3	:

	addi r3, label2

# 2 labels
label4:
label5:
	addi r4, label3

	addi r5, label4
	addi r6, label5

# reference label defined later
    addi r7, label6

label6:
    mov r1, r2
