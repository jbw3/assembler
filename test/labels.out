0101
b200
b302
b404
b506
b606
b70e
0112
